### PR TITLE
Layout Editor: we should not show the playlist editor for dynamic playlists

### PR DESCRIPTION
**Pull request by maurofmferrao**
relates to xibosignageltd/xibo-private#838

### CODE REVIEW

File: ui/src/layout-editor/main.js [hunk 1]
```diff
@@ -3609,6 +3609,13 @@ lD.openContextMenu = function(obj, position = {x: 0, y: 0}) {
     layoutObject.playlists.widgets.length > 0
   );
 
+  // Check if it's a dynamic playlist
+  layoutObject.isDynamicPlaylist =
+    (
+      layoutObject.isPlaylist &&
+      $(obj).find('.dynamic-playlist-icon').length === 1
```
Comment (dasgarner) [inline ui/src/layout-editor/main.js:3616]: Would it be better to add something more "permanent" to this DOM element? Perhaps a data attribute or similar. I worry that if we change the icon for some reason we create a hard-to-find bug?

Comment (maurofmferrao) [reply ui/src/layout-editor/main.js:3616]: Yeah, you're right. I will add a class to the main object and also for the convert playlist button on top. Thanks!

**Review by dasgarner — APPROVED** [review]

**Review by dasgarner — APPROVED** [review]

**Comment by ifarzana** [comment]
TEST: /Layout END

**Comment by ifarzana** [comment]
TEST: /Layout/Editor/layout_editor_dataset.cy.js END

**Pull request merged**